src/compas_view2/objects/pointobject.py

```diff
@@ -33,6 +34,10 @@ def init(self):
             'n': 1
         }
 
+    def edit(self, on_update=None):
```

Comment:
adding this edit function to each objects that will trigger its corresponding edit form. 
So externally it become simply `obj.edit()`